packages/app/src/admin/SuperAdminPage.tsx

```diff
@@ -180,7 +191,16 @@ export function SuperAdminPage(): JSX.Element {
           <Button type="submit">Get Database Stats</Button>
         </Stack>
       </Form>
-      <Modal opened={opened} onClose={close} title={modalTitle} centered>
+      <Divider my="lg" />
+      <Title order={2}>Database schema drift</Title>
```

Comment:
```suggestion
      <Title order={2}>Database Schema Drift</Title>
```
Small nit, prefer all first capitalized for consistency with other titles